tests/test_classification.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+from pyriemann.classification import TangentSpace
+from pyriemann.estimation import XdawnCovariances
+from pyriemann_qiskit.classification import QuanticSVM, QuanticVQC
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import StratifiedKFold, cross_val_score
+
+
+def test_GetSetParams(get_covmats, get_labels):
+    clf = make_pipeline(XdawnCovariances(), TangentSpace(),
+                        QuanticSVM(target=1, quantum=False))
+    skf = StratifiedKFold(n_splits=5)
+    n_matrices, n_channels, n_classes = 100, 3, 2
+    covset = get_covmats(n_matrices, n_channels)
+    labels = get_labels(n_matrices, n_classes)
+    cross_val_score(clf, covset, labels, cv=skf, scoring='roc_auc')
+
+
+def test_Quantic_init():
+    """Test init of quantum classifiers"""
+    ta = 1
+    # if "classical" computation enable,
+    # no provider and backend should be defined
+    q = QuanticSVM(target=ta, quantum=False)
+    q._init_quantum()
+    assert not q.quantum
+    assert not hasattr(q, "backend")
+    assert not hasattr(q, "provider")
+    # if "quantum" computation enabled, but no accountToken are provided,
+    # then "quantum" simulation will be enabled
+    # i.e., no remote quantum provider will be defined
+    q = QuanticSVM(target=ta, quantum=True)
+    q._init_quantum()
+    assert q.quantum
+    assert hasattr(q, "_backend")
+    assert not hasattr(q, "_provider")
+    # if "quantum" computation enabled, and accountToken is provided,
+    # then real quantum backend is used
+    # this should raise a error as uncorrect API Token is passed
+    try:
+        q = QuanticSVM(target=ta, quantum=True, qAccountToken="Test")
+        assert False  # Should never reach this line
+    except Exception:
+        pass
+
+
+def test_Quantic_splitTargetAndNonTarget(get_covmats, get_labels):
+    """Test _split_target_and_non_target method of quantum classifiers"""
+    n_matrices, n_channels, n_classes = 100, 3, 2
+    covset = get_covmats(n_matrices, n_channels)
+    labels = get_labels(n_matrices, n_classes)
+    q = QuanticSVM(target=1, quantum=False)
+    xta, xnt = q._split_target_and_non_target(covset, labels)
+    # Covariance matrices should be vectorized
+    class_len = n_matrices // n_classes  # balanced set
+    assert np.shape(xta) == (class_len, n_channels * n_channels)
+    assert np.shape(xnt) == (class_len, n_channels * n_channels)
+    # assert that the method still works
+    # when passing already vectorized matrices
+    covset.reshape(n_matrices, n_channels * n_channels)
+    try:
+        xta, xnt = q._split_target_and_non_target(covset, labels)
+        assert False  # Should never reach this line
+    except Exception:
+        pass
```

Comment:
I don't really understand this test, since QuanticSVM is supposed to process these vectorized inputs without problem.

```suggestion
    covset.reshape(n_matrices, n_channels * n_channels)
    xta, xnt = q._split_target_and_non_target(covset, labels)
    assert np.shape(xta) == (class_len, n_channels * n_channels)
    assert np.shape(xnt) == (class_len, n_channels * n_channels)
```

All public functions must be tested twice, with 3D inputs, and then with 2D vectorized inputs.